### PR TITLE
[Sentinel] [DataConnectors+ Entities] Live Validation Correctness fix

### PR DESCRIPTION
**Pull request by sagamzu**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [ ] update existing version for new feature 
      - [x] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [x] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by sagamzu** [comment]
@jianyexi  fixed the following swagger correctness issues:
image

**Comment by sagamzu** [comment]
@jianyexi  could you please take a look on this PR

**Comment by pilor** [comment]
Please go through the breaking change review process outlined in the PR description

**Comment by sagamzu** [comment]
> Please go through the breaking change review process outlined in the PR description

Don't we have an easy process for Fixs360 during “Window-to-Fix-Broken”?

**Comment by pilor** [comment]
> > Please go through the breaking change review process outlined in the PR description
> 
> Don't we have an easy process for Fixs360 during “[Window-to-Fix-Broken](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/405/Window-to-Fix-Broken)”?
> 
> Process change: When PR is labelled with "FixS360", breaking change can be approved by PR assignee; the Azure Breaking Change Board is no longer required to approve the PR. However, after the window period, all breaking changes must be presented & justified to the Azure Breaking Change Board according to Azure's policy: https://aka.ms/AzBreakingChangesPolicy .

I was not aware of this new tag.  Nevermind

**Comment by sagamzu** [comment]
> > > Please go through the breaking change review process outlined in the PR description
> > 
> > 
> > Don't we have an easy process for Fixs360 during “[Window-to-Fix-Broken](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/405/Window-to-Fix-Broken)”?
> > Process change: When PR is labelled with "FixS360", breaking change can be approved by PR assignee; the Azure Breaking Change Board is no longer required to approve the PR. However, after the window period, all breaking changes must be presented & justified to the Azure Breaking Change Board according to Azure's policy: https://aka.ms/AzBreakingChangesPolicy .
> 
> I was not aware of this new tag. Nevermind

thanks for the approval from your side. do you know who can complete this PR?
It is open for long time...

**Comment by sagamzu** [comment]
@jianyexi could you please look on this PR?

**Comment by jianyexi** [comment]
> @jianyexi Jianye Xi FTE could you please look on this PR?

Could you try to  fix the violation of CI checks ?

**Comment by sagamzu** [comment]
> > @jianyexi Jianye Xi FTE could you please look on this PR?
> 
> Could you try to fix the violation of CI checks ?

I cant fix most of them:

-  LintDiff failed -> this value define not as camel case in the code...

- cSpell failed -> 2 probelms:

   - syncer  - this is the word in the enum value, so I need to use it
   - MSTI - short for Microsoft Security Treat Inteligence. should I use the long name? 
   (I see that we can use MTP for example to short names)

**Comment by jianyexi** [comment]
> > > @jianyexi Jianye Xi FTE could you please look on this PR?
> > 
> > 
> > Could you try to fix the violation of CI checks ?
> 
> I cant fix most of them:
> 
> * LintDiff failed -> these values define **not** as camel case in the code...
> * cSpell failed -> 2 probelms:
>   
>   * syncer  - this is the word in the enum value, so I need to use it
>   * MSTI - short for Microsoft Security Treat Inteligence. should I use the long name?
>     (I see that we can use MTP for example to short names)

For cSpell failure, you can add the disallowed words into https://github.com/Azure/azure-rest-api-specs/blob/d35bfe73dd912ff9376bb0d425d27f8c821bdbab/custom-words.txt#L1

**Comment by sagamzu** [comment]
> > > > @jianyexi Jianye Xi FTE could you please look on this PR?
> > > 
> > > 
> > > Could you try to fix the violation of CI checks ?
> > 
> > 
> > I cant fix most of them:
> > 
> > * LintDiff failed -> these values define **not** as camel case in the code...
> > * cSpell failed -> 2 probelms:
> >   
> >   * syncer  - this is the word in the enum value, so I need to use it
> >   * MSTI - short for Microsoft Security Treat Inteligence. should I use the long name?
> >     (I see that we can use MTP for example to short names)
> 
> For cSpell failure, you can add the disallowed words into
> 
> https://github.com/Azure/azure-rest-api-specs/blob/d35bfe73dd912ff9376bb0d425d27f8c821bdbab/custom-words.txt#L1

Thanks!! done

**Comment by jianyexi** [comment]
@sagamzu  For the LintDiff errors, the property name should be camel case : dynamics365CDSActivities -> dynamics365CdsActivities

**Comment by sagamzu** [comment]
> @sagamzu For the LintDiff errors, the property name should be camel case : dynamics365CDSActivities -> dynamics365CdsActivities

@jianyexi fixed

**Comment by jianyexi** [comment]
@ArcturusZhang  could you take a look at Go SDK generation failure

**Comment by jianyexi** [comment]
@msyyc  could you take a look at the Python SDK failure?

File: specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json [hunk 1]
```diff
@@ -10744,6 +10972,160 @@
...
+        "Unknown",
+        "DeliveredAsSpam",
+        "Delivered",
+        "Blocked",
+        "Replaced"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "DeliveryAction",
```
Comment (ArcturusZhang) [inline specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json:11019]: We have two enum definitions with the same name, but different value set (one of them has 4 entries, and the other has 5). Could you please resolve this?

**Review by ArcturusZhang — CHANGES_REQUESTED** [review]

File: specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json (hunk 1 above)
Comment (sagamzu) [reply specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json]: Thanks! copy&paste mistake...
Fixed.

**Comment by sagamzu** [comment]
> @msyyc, @ArcturusZhang 
could you please look again on this PR?
it is open for a long time and block us from other changes (and to check that all the correctness issues fixed until the end of month)

**Comment by jianyexi** [comment]
@sagamzu  could you tell us which S360 Kpi you want to fix ?

**Comment by sagamzu** [comment]
> @jianyexi fixed the following swagger correctness issues:
> https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=CB57A901-1576-463C-B99B-031163D7B36D
> https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=5192A75F-060C-4CC4-ADE8-C5E41CDD6297
> https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=B31781BC-7FA3-4D11-9B01-C9695685D332
> https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=24023CD6-EF24-4B97-8A8C-379845044952
> https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=74DF9126-1C71-4BC8-9C96-728FE5312A23
> https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=60811810-B362-4CD5-A0DF-5CC2F38BEBF0
> https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=432BD763-3082-4CF9-A3A2-799F873F8384
> https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=12F145F7-15C8-4A86-B3BE-572E7D07F3B5
> https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=B80C30CC-0B27-4D2B-B53E-70074F33C90B
> https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=39FD433D-580D-4FC6-BE12-4E16CE3FF0BA
> https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=EE0EF90F-8250-45A0-8474-5D039B2D4A6C
> https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=64482E1E-3DAB-485D-B62E-99A964C451F9
> https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=ECF38145-ABD1-41E0-8983-B86210425D49
> https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=E05E09A5-1F04-402E-8353-E783FA314F41
> https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=776A570A-B6B5-4190-B483-4988B1D0B4F4

@jianyexi - these kpis. 
the PR is open for long time, so probably some were fix in another PRs (I had some merge)

**Comment by jianyexi** [comment]
@sagamzu  Do you fix the following issues ?
https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=EE0EF90F-8250-45A0-8474-5D039B2D4A6C
https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=64482E1E-3DAB-485D-B62E-99A964C451F9
 https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=12F145F7-15C8-4A86-B3BE-572E7D07F3B5 ?

**Comment by jianyexi** [comment]
> @sagamzu Do you fix the following issues ?
> https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=EE0EF90F-8250-45A0-8474-5D039B2D4A6C
> https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=64482E1E-3DAB-485D-B62E-99A964C451F9
> https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=12F145F7-15C8-4A86-B3BE-572E7D07F3B5 ?

Or can you provide the error list that you want to fix in this PR?

**Comment by sagamzu** [comment]
> > @sagamzu Do you fix the following issues ?
> > https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=EE0EF90F-8250-45A0-8474-5D039B2D4A6C
> > https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=64482E1E-3DAB-485D-B62E-99A964C451F9
> > https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=12F145F7-15C8-4A86-B3BE-572E7D07F3B5 ?
> 
> Or can you provide the error list that you want to fix in this PR?

In the above comment you can see the list, and as i mention maybe some of them already fixed in another PR because this one is open for a long time, and I did some merge

**Comment by sagamzu** [comment]
> > > @sagamzu Do you fix the following issues ?
> > > https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=EE0EF90F-8250-45A0-8474-5D039B2D4A6C
> > > https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=64482E1E-3DAB-485D-B62E-99A964C451F9
> > > https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=12F145F7-15C8-4A86-B3BE-572E7D07F3B5 ?
> > 
> > 
> > Or can you provide the error list that you want to fix in this PR?
> 
> In the above comment you can see the list, and as i mention maybe some of them already fixed in another PR because this one is open for a long time, and I did some merge

@jianyexi I go over the list again and these are the error List (some fixed in another PR):
https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=432BD763-3082-4CF9-A3A2-799F873F8384
https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=12F145F7-15C8-4A86-B3BE-572E7D07F3B5
https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=B80C30CC-0B27-4D2B-B53E-70074F33C90B
https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=776A570A-B6B5-4190-B483-4988B1D0B4F4
https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=E05E09A5-1F04-402E-8353-E783FA314F41
https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=ECF38145-ABD1-41E0-8983-B86210425D49
https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=39FD433D-580D-4FC6-BE12-4E16CE3FF0BA

**Comment by jianyexi** [comment]
@sagamzu  could you fix the duplicate schema errors :
️❌[Staging] SDK Track2 Validation: 8 Errors, 46 Warnings failed [Detail]

**Comment by sagamzu** [comment]
> @sagamzu could you fix the duplicate schema errors :
> ️❌[Staging] SDK Track2 Validation: 8 Errors, 46 Warnings failed [Detail]

@jianyexi I go over these errors and all of them are not related to the changes i did in this pr, so i am not going to fix them now. 
 i will schedule a meeting with the owner of the swagger on our side about these errors.

**Comment by jianyexi** [comment]
> > @sagamzu
> >             
> >             Sara Gamzu
> >             FTE could you fix the duplicate schema errors :
> > ️❌[Staging] SDK Track2 Validation: 8 Errors, 46 Warnings failed [Detail]
> 
> @jianyexi Jianye Xi FTE I go over these errors and all of them are not related to the changes i did in this pr, so i am not going to fix it now.
> i will schedule a meeting with the owner of the swagger on our side about these errors.

Actually it's easy to fix the issue , just rename the duplicate model name . see  https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/ci-fix.md#sdk-track2-validation on how to run locally.

**Review by jianyexi — APPROVED** [review]

**Review by ArcturusZhang — APPROVED** [review]

**Pull request merged**